Client/Frontend/Popup/CookieNotificationBlockingConsentView.swift

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import DesignSystem
+import BraveShared
+
+struct CookieNotificationBlockingConsentView: View {
+  public static let contentHeight = 480.0
+  private static let gifHeight = 328.0
+  private static let bottomSectionHeight = contentHeight - gifHeight
+  
+  private let animation = Animation.easeOut(duration: 0.5).delay(0)
+  private let transition = AnyTransition.scale(scale: 1.1).combined(with: .opacity)
+  private let textPadding: CGFloat = 16
+  
+  @Environment(\.presentationMode) var presentationMode
+  @State private var showAnimation = false
+  
+  var body: some View {
+    VStack {
+      VStack {
+        if !showAnimation {
+          VStack(alignment: .center, spacing: textPadding) {
+            Text(Strings.blockCookieConsentNoticesPopupTitle)
+              .font(.title)
+              .foregroundColor(Color(UIColor.braveLabel))
+              .multilineTextAlignment(.center)
+              .transition(transition)
+              .padding(.horizontal, textPadding)
```

Comment:
These 3 modifiers can be removed from individual views and put in the root VStack

```
.multilineTextAlignment(.center)
.transition(transition)
.padding(.horizontal, textPadding)
```